scripts/dist_build.sh

```diff
@@ -16,10 +16,15 @@ GIT_IMPORT="github.com/tendermint/tendermint/version"
 # Determine the arch/os combos we're building for
 XC_ARCH=${XC_ARCH:-"386 amd64 arm"}
 XC_OS=${XC_OS:-"solaris darwin freebsd linux windows"}
+XC_EXCLUDE=${XC_EXCLUDE:-" darwin/arm solaris/amd64 solaris/386 solaris/arm freebsd/amd64 windows/arm "}
 
 # Make sure build tools are available.
-# TODO: Tools should be "vendored" too.
-make get_tools
+curl -L https://github.com/Masterminds/glide/releases/download/v0.13.1/glide-v0.13.1-linux-amd64.tar.gz | tar xvfz - && \
+	pushd linux-amd64 && \
+	chmod +x glide && \
+	mv glide /usr/bin/ && \
```

Comment:
This is not going to fly in any environment. Even developers should not run their build scripts as administrator on the local machine. You have to understand that if this stays in the script, automation will avoid using this script completely.